config/environments/production.rb

```diff
@@ -65,4 +66,7 @@
   #Add support for asset compression in production
   # config.assets.css_compressor = :sass
   # config.assets.js_compressor = :uglifier
+
+  #Use for Opentelemtry tracing
+  config.tracer = OpenTelemetry.tracer_provider.tracer('my-tracer')
```

Comment:
I'm wondering if it makes more sense to move this logic (along with the same in development.rb) to the opentelemetry initializer